WalletWasabi.Fluent/ViewModels/CoinControl/Core/Cells/IndicatorsCellViewModel.cs

```diff
@@ -0,0 +1,23 @@
+namespace WalletWasabi.Fluent.ViewModels.CoinControl.Core.Cells;
+
+public class IndicatorsCellViewModel : ViewModelBase
```

Comment:
Can we keep same naming scheme for templates, everywhere else we use Columns, in this PR we use cells (both views and view models).

Reply:
Same as above.